core/src/main/java/org/fao/geonet/kernel/AccessManager.java

```diff
@@ -566,10 +566,17 @@ else if (ip.indexOf(':') >= 0) {
         try {
             if (network.isPresent() && netmask.isPresent() &&
                 StringUtils.isNotEmpty(network.get().getValue()) && StringUtils.isNotEmpty(netmask.get().getValue())) {
-                long lIntranetNet = getAddress(network.get().getValue());
-                long lIntranetMask = getAddress(netmask.get().getValue());
                 long lAddress = getAddress(ip.split(",")[0]);
-                return (lAddress & lIntranetMask) == (lIntranetNet & lIntranetMask);
+                String[] networkArray = network.get().getValue().split(",");
+                String[] netmaskArray = netmask.get().getValue().split(",");
+                for (int i = 0; i < networkArray.length; i++) {
+                    long lIntranetNet = getAddress(networkArray[i]);
+                    long lIntranetMask = getAddress(netmaskArray[i]);
```

Comment:
It may happen that the configuration contains fewer values ​​for `netmaskArray` than for `networkArray`.

I see 2 options:

1) Skip the network addresses that doesn't have a related network mask and log a warning with the issue

2) Apply to these network addresses the last network mask defined.

Please also update the help text describing the new format:

https://github.com/geonetwork/core-geonetwork/blob/2731e560fd9e3a8cea16047082cf557d3c0978c5/web-ui/src/main/resources/catalog/locales/en-admin.json#L703 to describe the updated format.

Reply:
Updated Jose, Intranet configuration expects to have same number of IP in network and netmask. If not an error is logged.
Help text updated accordingly.